apps/web/src/pages/brand/BrandPage.tsx

```diff
@@ -7,15 +7,18 @@ import PageHeader from '../../components/layout/components/PageHeader';
 import { useAuthContext } from '../../components/providers/AuthProvider';
 import { useSegment } from '../../components/providers/SegmentProvider';
 import { ROUTES } from '../../constants/routes.enum';
-import { colors, useTabsStyles } from '@novu/design-system';
-import { useEnvController } from '../../hooks';
+import { colors, useTabsStyles, When } from '@novu/design-system';
+import { useEnvController, useFeatureFlag } from '../../hooks';
+import { FeatureFlagsKeysEnum } from '@novu/shared';
 
-const BRANDING = 'Assets';
+const BRAND = 'Brand';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** I believe it's `Branding` for the new page ‏